From guillaume.bertholon@ens.fr  Tue Feb  1 18:50:20 2022
From: Guillaume Bertholon <guillaume.bertholon@ens.fr>
Date: Tue,  1 Feb 2022 18:11:13 +0100
Subject: Revert "drm/radeon/ci: disable mclk switching for high refresh rates (v2)"
To: gregkh@linuxfoundation.org
Cc: stable@vger.kernel.org, Guillaume Bertholon <guillaume.bertholon@ens.fr>
Message-ID: <1643735473-14635-1-git-send-email-guillaume.bertholon@ens.fr>

From: Guillaume Bertholon <guillaume.bertholon@ens.fr>

This reverts commit 0157e2a8a71978c58a7d6cfb3616ab17d9726631.

The reverted commit was backported and applied twice on the stable branch:
- First as commit 15de2e4c90b7 ("drm/radeon/ci: disable mclk switching for
high refresh rates (v2)")
- Then as commit 0157e2a8a719 ("drm/radeon/ci: disable mclk switching for
high refresh rates (v2)")

Fixes: 0157e2a8a719 ("drm/radeon/ci: disable mclk switching for high refresh rates (v2)")
Signed-off-by: Guillaume Bertholon <guillaume.bertholon@ens.fr>
Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>
---
 drivers/gpu/drm/radeon/ci_dpm.c |    6 ------
 1 file changed, 6 deletions(-)

--- a/drivers/gpu/drm/radeon/ci_dpm.c
+++ b/drivers/gpu/drm/radeon/ci_dpm.c
@@ -782,12 +782,6 @@ bool ci_dpm_vblank_too_short(struct rade
 	if (r600_dpm_get_vrefresh(rdev) > 120)
 		return true;
 
-	/* disable mclk switching if the refresh is >120Hz, even if the
-        * blanking period would allow it
-        */
-	if (r600_dpm_get_vrefresh(rdev) > 120)
-		return true;
-
 	if (vblank_time < switch_limit)
 		return true;
 	else
